Reading from Hadoop

WheresAlice opened the pull request:
I'm not quite sure what to title this, and whether to target big data people or engineers in general.  But here's an initial draft of my next post for visibility.  I can add some more graphs when I'm back in the office tomorrow.

glenjamin commented:
:+1: - might be good to make each initial mention of a Hadoop-related technology into a link to the project page.


Title perhaps "When Hadoop tools disagree with each other" ?

WheresAlice commented:
I think I'm happy with this now I've added graphs and incorporated changes suggested by Glen and Darrell.  Reviews from non-data people would be welcome.

medains commented on _posts/2015-12-04-when-hadoop-tools-disagree.md:
```diff
@@ -0,0 +1,43 @@
+
+![Graph of 8-year spike](/images/8yearspike.png)
+
+The above graph from our datawarehouse system was recently tweeted out, and got some amused responses about 2007 wanting its data back.  But there's actually a good story here, and a chance to look at Hadoop's dirty secrets.
```
Make "datawarehouse" two words

medains commented on _posts/2015-12-04-when-hadoop-tools-disagree.md:
```diff
@@ -0,0 +1,43 @@
+
+But if we zoom in on that flat portion of the first graph, we don't see a nice clean sawtooth graph, it's actually rather jagged.  To explain that lets look at the process it's graphing.
+
+Data is [sqooped](http://sqoop.apache.org/) into HDFS from a standard RDBMS.  It's then processed in [Hive](http://hive.apache.org/), and finally we perform an insert overwrite into the production Hive table.  But this graph is generated using [Impala](http://impala.io/), the interface that our BI tool uses.  In order for Impala to know about changed data in a Hive table, you need to perform a re ...
```
combine the first two sentences with a ; instead of separating with a .

medains commented on _posts/2015-12-04-when-hadoop-tools-disagree.md:
```diff
@@ -0,0 +1,43 @@
+
+And this is the dirty secret.  Whilst you can read HDFS data using Hive, [Spark](http://spark.apache.org/), Impala, [Pig](http://pig.apache.org/), etc. Unless you're using the same tool as the writer, you're bound to hit this sort of problem pretty frequently.
+
+So how do we resolve it?
```
Make this some kind of heading

medains commented:
Title may be better to be "Hadoop's dirty little secret" since you mention that a couple of times, and it's more linkbaity :)

WheresAlice commented:
I feel wrong using that phrase, I'm not sure I can bring myself to turn this into full-on click bait. I've made the other changes you've suggested though.

adamhepton commented:
Can we put this on early next week so it doesn't get lost over the weekend?

WheresAlice commented:
Yeah OK, give me a publish time and date and I'll update the pull request.

glenjamin commented:
I wonder if we could use zapier to schedule this

WheresAlice commented:
I've bumped the date of this to tomorrow.  Please can this be published now?
